__tests__/__snapshots__/Transactions.Landscape.snapshot.tsx.snap

```diff
@@ -0,0 +1,727 @@
+            onResponderGrant={[Function]}
+            onResponderMove={[Function]}
+            onResponderRelease={[Function]}
+            onResponderTerminate={[Function]}
+            onResponderTerminationRequest={[Function]}
+            onStartShouldSetResponder={[Function]}
+            style={
+              {
+                "opacity": 1,
+              }
+            }
+          >
+            <View
+              style={
+                {
+                  "borderBottomColor": "rgb(216, 216, 216)",
+                  "borderBottomWidth": 1,
+                  "display": "flex",
+                  "flexDirection": "row",
+                  "marginTop": 15,
+                  "paddingBottom": 15,
+                }
+              }
+            >
+              <
+                color="rgb(0, 122, 255)"
+                icon={
+                  {
+                    "icon": [
+                      384,
+                      512,
+                      [
+                        8595,
+                      ],
+                      "f063",
+                      "M169.4 470.6c12.5 12.5 32.8 12.5 45.3 0l160-160c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L224 370.8 224 64c0-17.7-14.3-32-32-32s-32 14.3-32 32l0 306.7L54.6 265.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l160 160z",
+                    ],
+                    "iconName": "arrow-down",
+                    "prefix": "fas",
+                  }
+                }
+                size={24}
+                style={
+                  {
+                    "marginLeft": 5,
+                    "marginRight": 5,
+                    "marginTop": 5,
+                  }
+                }
+              />
+              <View
+                style={
+                  {
+                    "display": "flex",
+                  }
+                }
+              >
+                <Text
+                  style={
+                    {
+                      "color": "rgb(28, 28, 30)",
+                      "fontSize": 18,
+                      "opacity": 0.65,
+                    }
+                  }
+                >
+                  Unknown
+                </Text>
+                <View
+                  style={
+                    {
+                      "display": "flex",
+                      "flexDirection": "row",
+                    }
+                  }
+                >
+                  <Text
+                    style={
+                      {
+                        "color": "rgb(28, 28, 30)",
+                        "opacity": 0.65,
+                      }
+                    }
+                  >
+                    Sent 
+                  </Text>
+                  <Text
+                    style={
+                      {
+                        "color": "rgb(28, 28, 30)",
+                        "opacity": 0.65,
+                      }
+                    }
+                  >
+                    Jan 1, 8:48 pm
```

Comment:
As above, this fails as it outputs 11:48 pm on my computer due to time zones.

Reply:
same problem.